core/src/test/kotlin/cash/grammar/kotlindsl/utils/DependencyExtractorTest.kt

```diff
@@ -61,8 +61,8 @@ internal class DependencyExtractorTest {
         fullText = "api(libs.magic)",
       )
     )
-    assertThat(scriptListener.expressions).containsExactly("add(\"extraImplementation\", libs.fortyTwo)")
```

Comment:
Expressions used to contain a String of the leaf PostfixUnaryExpressionContext that is NOT a declaration. Moving this to statements since we group all non declaration to `StatementContext`. 

If we still need to know this for sorter, we can introduce a wrapper type to give this more meaning.

Reply:
Nah this is perfect. Thanks!